Cargo.toml

```diff
@@ -13,5 +13,8 @@ members = [
     "sm2"
 ]
 
+[patch.crates-io]
+sp1-lib = { git = "https://github.com/succinctlabs/sp1.git", branch = "n/fp_ops_hook" }
```

Comment:
Add a TODO that this needs to be switched back to SP1 once the release with the hooks is complete.